Assets/SteamVR_Unity_Toolkit/Scripts/SteamVR_ControllerEvents.cs

```diff
@@ -28,13 +28,15 @@ public enum ButtonAlias
     public int axisFidelity = 1;
 
     public bool triggerPressed = false;
+    public bool triggerDown = false; //Seems to work more reliably than triggerPressed? Added 5/24/2016 - Blueteak
```

Comment:
This is a bit of an anomaly, The way triggerDown is being set is not really that much different to how triggerPressed is being set.

the `OnTriggerClicked` method is called by passing the results of the `SetButtonEvent` method and that method sets the boolean.

There "should" be no difference between the two variables.